tasks/Module_2/task_1.py

```diff
@@ -0,0 +1,17 @@
+stuff = {'rope': 1, 'torch': 6, 'gold coin': 42, 'dagger': 1, 'arrow': 12}
+def display_inventory(inventory):
+    print("Inventory:")
+    item_total = 0
+    overload_warning =''
+    for item_name, item_count in inventory.items():
+        item_total += item_count
+        print(str(item_count) + ' ' + item_name)
+    print("\nTotal number of items: " + str(item_total)+'\n')
```

Comment:
As an improvement, as we are using Python 3.7 or newer, we can make good usage out of fstrings.
They improve readability and simplify things a lot.

https://realpython.com/python-f-strings/

But that's optional and up to you